config/default.yml

```diff
@@ -262,3 +262,14 @@ CommitMsg:
 
   TrailingPeriod:
     description: 'Checking for trailing periods in subject'
+
+PostCommit:
+  ALL:
+    requires_files: false
+    required: false
+    quiet: false
+
+  GitGuilt:
```

Comment:
This is a cute tool, but I don't see the functionality it provides as something that most developers would find useful to have by default. It doesn't speed up any particular process (that I'm aware of) and appears to be a vanity metric for the most part (feel free to give some concrete examples of where this would be useful in the general case).

Please add `enabled: false` so this does not run by default.